src/app/layout.tsx

```diff
@@ -1,21 +1,15 @@
 import "./globals.css";
 import { Inter } from "next/font/google";
 import { OIDCProvider } from "@/components/auth/OIDCUtils";
-import { OidcConfiguration } from "@axa-fr/react-oidc";
 
 const inter = Inter({ subsets: ["latin"] });
 
 export const metadata = {
-  title: "diracX",
+  title: "DiracX",
   description: "Distributed Infrastructure with Remote Agent Controller",
 };
 
-const configuration: OidcConfiguration = {
-  client_id: process.env.DIRACX_CLIENT_ID || "",
-  redirect_uri: process.env.REDIRECT_URI || "",
-  scope: process.env.DEFAULT_SCOPE || "",
-  authority: process.env.NEXT_PUBLIC_DIRACX_URL || "",
-};
+export const dynamic = "error";
```

Comment:
What is the purpose of this line?

Reply:
I think it causes `npm run dev` to error when you accidentally use features that aren't possible to use with the static export.

See these two:

* https://nextjs.org/docs/app/building-your-application/deploying/static-exports#unsupported-features
* https://nextjs.org/docs/app/api-reference/file-conventions/route-segment-config#dynamic

Reply:
I've added a comment about it 😄